### DISCUSSION TITLE
Check the url by name and direct it to a specific route?

### DISCUSSION THREAD

**Discussion by lucasjrnany** [dotnet/yarp#2542] [Q&A] [answered]
I have some end points, I need to check if they contain the word "Sales" and redirect them to the server that will process them!

localhost:5001//api/v1/SalesPayment
localhost:5001//api/v1/SalesOrder
localhost:5001//api/v1/SalesStock

A way to do it via middleware or another way, so I don't have to declare route by route because I have several end points, I wanted to process different servers.

**Comment by Tratcher** [comment, marked as answer]
https://microsoft.github.io/reverse-proxy/articles/config-files.html#routes
> Path is an ASP.NET Core route template that can be defined as [explained here](https://docs.microsoft.com/aspnet/core/fundamentals/routing#route-templates).
